app/controlplane/internal/biz/robotaccount_integration_test.go

```diff
@@ -0,0 +1,85 @@
+//
+// Copyright 2024 The Chainloop Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package biz_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/chainloop-dev/chainloop/app/controlplane/internal/biz"
+	"github.com/chainloop-dev/chainloop/app/controlplane/internal/biz/testhelpers"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+func (s *robotAccountTestSuite) TestRevoke() {
+	ctx := context.Background()
+	s.Run("returns an error if org ID format is not valid", func() {
+		err := s.RobotAccount.Revoke(ctx, "not_valid_uuid", "52597CC8-21BF-45C4-B30B-AE2879556CE1")
+		s.ErrorAs(err, &biz.ErrInvalidUUID{})
+	})
+
+	s.Run("returns an error if robot account ID format is not valid", func() {
+		err := s.RobotAccount.Revoke(ctx, "991B156A-7636-44D2-97B0-E97070E626DC", "not_valid_uuid")
+		s.ErrorAs(err, &biz.ErrInvalidUUID{})
+	})
+
+	s.Run("returns a Not Found if robot account cannot be found", func() {
+		err := s.RobotAccount.Revoke(ctx, s.org.ID, "DCF3895A-22BE-49D0-99D4-0A76835D0BD1")
```

Comment:
in other places, instead of using a hardcoded UUID I use uuid.NewString(). 

I am curious to know why/if you prefer your approach